watertap/unit_models/MD/membrane_distillation_0D.py

```diff
@@ -0,0 +1,567 @@
+        "pressure_change_type",
+        ConfigValue(
+            default=PressureChangeType.fixed_per_stage,
+            domain=In(PressureChangeType),
+            description="Pressure change term construction flag",
+            doc="""
+        Indicates what type of pressure change calculation will be made. To use any of the
+        ``pressure_change_type`` options to account for pressure drop, the configuration keyword
+        ``has_pressure_change`` must also be set to ``True``. Also, if a value is specified for pressure
+        change, it should be negative to represent pressure drop.
+
+        **default** - ``PressureChangeType.fixed_per_stage`` 
+
+        .. csv-table::
+            :header: "Configuration Options", "Description"
+        
+            "``PressureChangeType.fixed_per_stage``", "Specify an estimated value for pressure drop across the membrane feed channel"
+            "``PressureChangeType.fixed_per_unit_length``", "Specify an estimated value for pressure drop per unit length across the membrane feed channel"
+            "``PressureChangeType.calculated``", "Allow model to perform calculation of pressure drop across the membrane feed channel"
+        """,
+        ),
+    )
+
+    _CONFIG_Template.declare(
+        "friction_factor",
+        ConfigValue(
+            default=FrictionFactor.flat_sheet,
+            domain=In(FrictionFactor),
+            description="Darcy friction factor correlation",
+            doc="""
+            Options to account for friction factor correlations.
+
+            **default** - ``FrictionFactor.flat_sheet`` 
+
+        .. csv-table::
+            :header: "Configuration Options", "Description"
+
+            "``FrictionFactor.flat_sheet``", "Friction factor correlation for flat-sheet membrane modules"
+            "``FrictionFactor.spiral_wound``", "Friction factor correlation for spiral-wound membranes"
+        """,
+        ),
+    )
+
+    _CONFIG_Template.declare(
+        "property_package_vapor",
+        ConfigValue(
+            default=useDefault,
+            domain=is_physical_parameter_block,
+            description="Property package to use for control volume",
+            doc="""Property parameter object used to define property calculations,
+        **default** - useDefault.
+        **Valid values:** {
+        **useDefault** - use default package from parent model or flowsheet,
+        **PhysicalParameterObject** - a PhysicalParameterBlock object.}""",
+        ),
+    )
+    _CONFIG_Template.declare(
+        "property_package_args_vapor",
+        ConfigBlock(
+            implicit=True,
+            description="Arguments to use for constructing property packages",
+            doc="""A ConfigBlock with arguments to be passed to a property block(s)
+        and used when constructing these,
+        **default** - None.
+        **Valid values:** {
+        see property package for documentation.}""",
+        ),
+    )
+
```

Comment:
Is there any need to have `property_package_vapor` specified for each channel, or can we just specify it once on the main `CONFIG` block?

Reply:
From the tests, I see you need to specify twice (once for each channel). Maybe there is some alternative configuration where you might want to consider 2 different vapor packages that I am not seeing, but from my current perspective, I don't see any immediate use cases for having 2 different vapor property packages.